torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10831,7 +10764,8 @@ def ref_pairwise_distance(input1, input2):
     OpInfo('svd',
            op=torch.svd,
            dtypes=floating_and_complex_types(),
-           sample_inputs_func=sample_inputs_svd,
+           sample_inputs_func=partial(sample_inputs_svd, is_linalg_svd=False),
```

Comment:
Note to self, I should just check the `op_info` inside the generator rather than passing a flag.

Reply:
Either pattern is fine